clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -115,28 +115,55 @@ public void disconnected(String id, long now) {
     }
 
     /**
+     * Enter the checking_api_versions state for the given node.
+     * @param id The connection identifier.
+     */
+    public void checkingApiVersions(String id) {
```

Comment:
Looks like only the consumer and producer packages are considered API: https://kafka.apache.org/090/javadoc/index.htm